SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -73,20 +73,15 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 
 void USpatialGameInstance::CreateNewSpatialWorkerConnection()
```

Comment:
Should this be renamed to `CreateNewSpatialWorkerConnectionManager` now?

Reply:
Good point, done.